### DISCUSSION TITLE
Cascader多选模式设置一个默认值(不考虑一个以上)，下拉框内容可以展示多级的列吗？

### DISCUSSION THREAD

**Discussion by i11I04i** [ant-design/ant-design#43052] [Q&A]
正常Cascader单选模式有默认值，下拉框会展示默认值的所有层级。希望多选模式下单个默认值能有一样的效果，大佬们有啥解决方案吗？antd4版本

**Comment by zombieJ** [comment]
是长啥样子？

**Reply by i11I04i** [reply]
![image](https://github.com/ant-design/ant-design/assets/58681207/c0284ff0-abb6-4098-88a3-9cd8a8a87f8c)
![image](https://github.com/ant-design/ant-design/assets/58681207/c4840d11-008c-44dd-a55e-9349475a66f5)
期望就是后者点击效果如前者
